### PR TITLE
[Premium Files Preview] Made the following cmdlets changes 

### PR DESCRIPTION
**Pull request by arkrasik**
- Set-PowerBIDataset that initiates abf <-> premium files migration
- Get-PowerBIWorkspaceMigrationStatus to check migrations statuses

### CODE REVIEW

File: src/Modules/Data/Commands.Data/GetPowerBIDataset.cs [hunk 1]
```diff
@@ -53,6 +53,16 @@ public class GetPowerBIDataset : PowerBIClientCmdlet, IUserScope, IUserFilter, I
...
         public int? First { get; set; }
 
+        [Alias("Expand")]
+        [Parameter(Mandatory = false, ParameterSetName = IdParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = NameParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ListParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectIdParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectNameParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectListParameterSetName)]
+        [ValidateSet("actualStorage")]
```
Comment (CodeCyclone) [inline src/Modules/Data/Commands.Data/GetPowerBIDataset.cs]: Make capital and you might need to do empty string too, unless validateset only runs when something is specified

File: src/Modules/Data/Commands.Data/SetPowerBIDataset.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+        [Parameter(Mandatory = true, ParameterSetName = DatasetNameParameterSetName)]
+        [Parameter(Mandatory = true, ParameterSetName = ObjectNameParameterSetName)]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = DatasetIdParameterSetName)]
+        [Parameter(Mandatory = true, ParameterSetName = DatasetNameParameterSetName)]
+        [Parameter(Mandatory = true, ParameterSetName = ObjectIdParameterSetName)]
+        [Parameter(Mandatory = true, ParameterSetName = ObjectNameParameterSetName)]
+        [ValidateSet("Abf", "PremiumFiles")]
```
Comment (CodeCyclone) [inline src/Modules/Data/Commands.Data/SetPowerBIDataset.cs]: What is Abf? Might need to spell out and then convert unless this is well documented

File: src/Modules/Data/Commands.Data/GetPowerBIDataset.cs (hunk 1 above)
Comment (CodeCyclone) [reply src/Modules/Data/Commands.Data/GetPowerBIDataset.cs]: ValidateSet only runs when specified, ignore that. Just make capital

Comment (arkrasik) [reply src/Modules/Data/Commands.Data/GetPowerBIDataset.cs]: 1) and lowercase it for request? i mean -- the request needs lcfirst
2)
```
> (Get-PowerBIDataset -Id 9139b36f-e02c-46dd-bfdd-224f757a6a28 -Include actualStorage -Scope Organization).Id

Guid
----
9139b36f-e02c-46dd-bfdd-224f757a6a28


> (Get-PowerBIDataset -Id 9139b36f-e02c-46dd-bfdd-224f757a6a28 -Scope Organization).Id

Guid
----
9139b36f-e02c-46dd-bfdd-224f757a6a28

```

looks ok

File: src/Modules/Data/Commands.Data/SetPowerBIDataset.cs (hunk 2 above)
Comment (arkrasik) [reply src/Modules/Data/Commands.Data/SetPowerBIDataset.cs]: Checked -- it's ok

**Pull request merged**